Mathlib/LinearAlgebra/RootSystem/Basic.lean

```diff
@@ -299,7 +172,8 @@ its roots. -/
 protected lemma ext [CharZero R] [NoZeroSMulDivisors R M]
     {P₁ P₂ : RootSystem ι R M N}
     (he : P₁.toLin = P₂.toLin)
-    (hr : P₁.root = P₂.root) :
+    (hr : P₁.root = P₂.root)
+    (hp : P₁.reflection_perm = P₂.reflection_perm) :
```

Comment:
We shouldn't need the `hp` argument, right?

I think we may need to reorganise slightly so that we don't get a strictly weaker `ext` lemma here.

Reply:
Yes, that's right.  I will try to fix this.

Reply:
Okay, I think I got it.

Reply:
Great, thanks!